dtests/src/test/java/io/snappydata/hydra/storeTests/sqlDisk/sqlDisk.bt

```diff
@@ -173,4 +173,4 @@ sql/sqlDisk/randomPartitionReplSyncOfflinePersistTablesHA.conf
     dropIndex = true
     redundantCopies=2
     workIterationsPerThread = 5
-*/
```

Comment:
Remove commented tests from bt file.

Reply:
we may need to visit them when we start testing HA, hence kept it for reference.